types/react-overlays/lib/RootCloseWrapper.d.ts

```diff
@@ -17,6 +23,4 @@ interface RootCloseWrapperProps {
 	event?: 'click' | 'mousedown';
 }
 
-declare class RootCloseWrapper extends React.Component<RootCloseWrapperProps> { }
-declare namespace RootCloseWrapper { } // module export workaround: https://github.com/Microsoft/TypeScript/issues/5073
-export = RootCloseWrapper;
+export default class RootCloseWrapper extends React.Component<RootCloseWrapperProps> { }
```

Comment:
Ah, it looks like it doesn't. Sorry I should have tested it more thoroughly. I'll fix this up tomorrow and will likely need to do the same for #18192. With that namespace hack, individual components will need to be imported with the `import * as ` syntax, right?

Is there really no other way to get interop between TS and babel's default exports without so much boilerplate?